packages/core/src/codecatalyst/devEnv.ts

```diff
@@ -205,10 +210,6 @@ export class InactivityMessage implements vscode.Disposable {
     }
 
     private clearOldMessage() {
-        if (this.#nextWholeMinute) {
-            clearTimeout(this.#nextWholeMinute)
-            this.#nextWholeMinute = undefined
-        }
         if (this.#beforeMessageShown) {
```

Comment:
TODO: `#beforeMessageShown` is cleared by `this.message()` and never re-initialized. Is that intended? @nkomonen-amazon